src/main/resources/db/migration/V1_132__print_files_to_clob_update.sql

```diff
@@ -0,0 +1,216 @@
+DROP PROCEDURE juror_mod.printfiles_to_clob(int4);
+
+CREATE OR REPLACE PROCEDURE juror_mod.printfiles_to_clob(IN p_document_limit integer DEFAULT 1000000)
+ LANGUAGE plpgsql
+AS $procedure$
+DECLARE
+	v_form_type VARCHAR(6);
+	v_max_rec_len INTEGER;
+	v_ext_date DATE;
+	forms RECORD;
+	v_count INTEGER:=0;
+
+BEGIN
+	v_ext_date :=	CASE
+						WHEN TO_CHAR(NOW(),'SSSSS')::INTEGER < 64800
+              				THEN NOW()::date - 1
+               				ELSE NOW()
```

Comment:
Don't we need to cast to a date as is done in the previous line?

Reply:
Added comment to explain as to why not